docs/tutorials/using_apis/Mbed_USB_WAV_Audio_Player/Mbed_USB_WAV_Audio_Player.md

```diff
@@ -0,0 +1,88 @@
+# Mbed USB WAV Audio Player
+This is a tutorial on how to put together a USB WAV audio player with Mbed. In general, the user will load a WAV file onto an SD card that will be connected to an Mbed board. The user can then play the WAV file from the board on a host computer over USB. Once connected, the Mbed board will act as a USB microphone that will be playing the sound, so that the sound can be looped back to speakers, headphones, or recorded with a program like Audacity.
+
+### Libraries
+This tutorial uses several Mbed libraries:
+- USB (part of Mbed-OS)
+- [sd-driver](https://github.com/ARMmbed/sd-driver) (separate Mbed library)
+- [AudioPlayer](https://github.com/c1728p9/AudioPlayer) (separate Mbed library)
+
+The USBAudio class of USB library is used to send audio data from the Mbed board to the host PC. The SDBlockDevice and FATFileSystem classes of sd-driver library are used so that WAV files can be stored on an SD card and accessed by the Mbed board. Lastly, the AudioPlayer and WaveAudioStream classes of the AudioPlayer library are used to access the audio data from the WAV file on the SD card.
+
+### What You Need
+- Mbed Enabled board
+- SD card
+- SD card module (if board does not already have one)
+- USB cable
+
+### Setup
+The following steps demonstrate the setup and use of the Mbed WAV Audio Player:
+1. Create a new Mbed project.   
+`mbed new <project name>`
+2. Add the [sd-driver](https://github.com/ARMmbed/sd-driver) and [AudioPlayer](https://github.com/c1728p9/AudioPlayer) libraries into the new Mbed project directory.   
+`mbed add https://github.com/ARMmbed/sd-driver`    
+`mbed add https://github.com/c1728p9/AudioPlayer`
+3. Copy and paste the below example code into *main.cpp*.
+4. Load WAV file(s) onto SD card. The example below uses a public domain WAV file called "Bach-minuet-in-g.wav" (attached below for download) that is inside of a "songs" directory on the SD card. It is important that the WAV file be PCM signed 16-bit little endian, or else it will not play. The WAV file can have any sampling rate and can have any number of channels. [Online-convert](https://audio.online-convert.com/convert-to-wav) can be used to achieve WAV files with different formats from any source audio file.   
+![N](https://raw.githubusercontent.com/mrcoulter45/mbed-os-5-docs/Mbed_USB_WAV_Audio_Player_Tutorial/docs/tutorials/using_apis/Mbed_USB_WAV_Audio_Player/images/Mbed_USB_WAV_Audio_Player_img1.png)
+5. Compile the program and flash the binary.
+6. Ensuring that the USBPhy is connected to the host computer, select "Mbed Audio" as the host PC's default audio input device. On Windows, in Control Panel > Hardware and Sound > Sound, in the recording tab, right click on "Microphone Mbed Audio" > Properties > Listen. Check the "Listen to this Device" box and click Apply. The audio from the WAV file should now be audible through whichever output device is selected for the host PC's audio output.   
```

Comment:
It may be clearer if you replace `the USBPhy is connected` with `the target's USB is connected` in this sentence.